Assets/MixedRealityToolkit.SDK/Features/Utilities/Solvers/SolverHandler.cs

```diff
@@ -408,21 +384,28 @@ protected virtual void AttachToNewTrackedObject()
                 if (TrackedHandness == Handedness.Both)
                 {
                     currentTrackedHandedness = PreferredTrackedHandedness;
-                    target = GetControllerRay(currentTrackedHandedness);
-                    if (target == null)
+                    controllerRay = PointerUtils.GetPointer<LinePointer>(currentTrackedHandedness);
```

Comment:
Will this work if a developer has removed line pointers from their pointer profile but still wants to use solvers? I actually do this sometimes when I don't want far interaction.

Reply:
Ah, answered my own question -- I suppose if you are using the controller ray, then you likely are using Line Pointers or something that extends linePointer

Reply:
Exactly, you have to have pointers if you are using Controller Ray